### PR TITLE
better updates to ScatterView classes

### PR DESCRIPTION
**Pull request by jreus**
updated ScatterView classes for Qt (on SC 3.9.3)
also added a grid to ScatterView and some small additional styling options

### CODE REVIEW

**Comment by LFSaw** [comment]
why did you open a new PR? You can just push to the branch of the old PR... 
That would make the process more transparent...

**Comment by telephon** [comment]
@LFSaw can you briefly check if this can be merged (finally!)?

**Comment by LFSaw** [comment]
if you want me to review this, please create a custom branch with a name _not_ being *master* since this makes my git ecosystem choke...

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.DS_Store
```
Comment (LFSaw) [inline .gitignore]: please remove .DS_Store

File: classes/Datasets/Transforms.sc [hunk 2]
```diff
@@ -0,0 +1,122 @@
+/*****************************************
+Data Transforms
+(C) 2018 Jonathan Reus
```
Comment (LFSaw) [inline classes/Datasets/Transforms.sc:3]: should this be GPL?

File: classes/Datasets/Transforms.sc [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+/*
+TO STANDARDIZE:
+1. calculate mean and standard deviation of each feature
+2. subtract mean from each feature
+3. divide features by standard deviation
+
+T is for Transform
+*/
+TStandardizer {
```
Comment (LFSaw) [inline classes/Datasets/Transforms.sc:83]: are these doing anything?

File: classes/various/Matrix.sc [hunk 4]
```diff
@@ -146,7 +146,7 @@ Matrix[slot] : Array {
 		},{
 			(vals.size).min(this.cols).do({ arg col;
 				this.put(row, col, vals.at(col))
-			});"Warning: wrong number of vals".postln;
+			});"Warning: wrong number of vals: % received and % expected".format(vals.size, this.cols).postln;
```
Comment (LFSaw) [inline classes/various/Matrix.sc:175]: I'd prefer these in new lines if possible

File: classes/various/Matrix.sc [hunk 5]
```diff
@@ -157,7 +157,7 @@ Matrix[slot] : Array {
 		},{
 			(vals.size).min(this.rows).do({ arg row;
 				this.put(row, col, vals.at(row))
-			});"Warning: wrong number of vals".postln;
+			});"Warning: wrong number of vals: % received and % expected".format(vals.size, this.cols).postln;
```
Comment (LFSaw) [inline classes/various/Matrix.sc:186]: I'd prefer these in new lines if possible

**Review by LFSaw — COMMENTED** [review]
please remove

**Comment by telephon** [comment]
@jreus could you please rename the branch? "master" is indeed not a good name! :)

**Review by patrickdupuis — DISMISSED** [review]
the copyright needs to be removed from this PR before it can be merged.

File: classes/Datasets/Transforms.sc (hunk 2 above)
Comment (patrickdupuis) [reply classes/Datasets/Transforms.sc:3]: Should also add Jonathan Reus as a contributor in the README.

**Comment by jreus** [comment]
@telephon Sorry for the long delay! I made a new branch called 'scatterplot' and submitted a pull request. Also resolved the .DS_Store conflict in .gitignore

**Comment by LFSaw** [comment]
closed in favour of #30

**Pull request closed without merging**